docs/10_dart_code_style.md

```diff
@@ -92,28 +92,125 @@ be started in [Intellij](https://youtrack.jetbrains.com/issue/WEB-44842)
     `Good:`
             
     ```dart
-    test(".fromJson() should create an instance from the json map", () {
+    test(".fromJson() creates an instance from the json map", () {
     ```
    
-   `Bad:`
+    `Bad:`
                
-   ```dart
-   test('.fromJson() should create an instance'
+    ```dart
+    test('.fromJson() creates an instance'
         'from the json map', () {
-   ```
+    ```
 8. PREFER using double quotes in test descriptions
     
     `Good:`
     
     ```dart
-    test(".fromJson() should create an instance from the json map", () {
+    test(".fromJson() creates an instance from the json map", () {
     ```
+9. DO NOT use "should" in test descriptions
+    
+    `Good:`
+         
+    ```dart
+    test(".fromJson() creates an instance from the json map", () {
+    ```
+    
+    `Bad:`
+              
+    ```dart
+    test(".fromJson() should create an instance from the json map.", () {
+    ```
+    Additional information: 
+    - https://enterprisecraftsmanship.com/posts/you-naming-tests-wrong/
+    - https://medium.com/frontmen/remove-the-verb-should-from-your-unit-test-descriptions-please-ee3a58f3dda0
+    
+10. DO NOT document Mock classes
+    
+    `Good:`
+    
+    ```dart
+    class SourceClientMock extends Mock implements SourceClient {}
+    ```
+    
+    `Bad:`
+              
+    ```dart
+    /// This class mocks [SourceClient] to help with testing.
+    class SourceClientMock extends Mock implements SourceClient {}
+    ```
+    
+11. DO document Mock static fields
```

Comment:
```suggestion
11. DO document Mock static fields
     Though it is assumed that Mock classes do not have any fields, sometimes it appears to be very pragmatic to add some _static_ fields to them in order to simplify code and tests structure, follow DRY or KISS, etc. (see Dart Mockito [best practices](https://pub.dev/packages/mockito#best-practices))
     
    `Good:`

    ```dart
    class CoolClassMock extends Mock implements CoolClass {
    /// The really useful field that allows doing something extra-cool 
    /// in order to prevent something very bad.
    static const String reallyUsefulField = 'Really Useful Value';
    ```

    `Bad:`

    ```dart
    class CoolClassMock extends Mock implements CoolClass {
    static const String reallyUsefulField = 'Really Useful Value';
    ```
```

It seems like we should follow the same style for each rule we're adding. Hence, even if we don't have a real example in it, a developer should always be able to follow a pattern.